api/src/main/java/org/ccci/idm/user/DefaultUserManager.java

```diff
@@ -307,4 +337,28 @@ protected void validateUser(final User user) throws UserException {
             throw new InvalidEmailUserException("Invalid email specified for user");
         }
     }
+
```

Comment:
I like having this interface defined in `DefaultUserManager` since it is only used for `DefaultUserManager` and not elsewhere.

It seemed like a good quick place to define the abstract class as well for similar reasons.